docs/source/format/CanonicalExtensions.rst

```diff
@@ -251,6 +251,14 @@ Variable shape tensor
    Values inside each **data** tensor element are stored in row-major/C-contiguous
    order according to the corresponding **shape**.
 
+UUID
+====
```

Comment:
I wonder if we should add a small note that we do not require any specific version of UUID.  I.e. if a field is a UUID field then the user has no guarantee that it is UUIDv4 or v1 or it could even be mixed.

Reply:
Good point. Added:
```

.. note::
   A specific UUID version is not required or guaranteed. This extension represents
   UUIDs as FixedSizeBinary(16) and does not interpret the bytes in any way.
```